packages/service-library/src/servicelib/tracing.py

```diff
@@ -0,0 +1,33 @@
+from typing import TypeAlias
+
+from opentelemetry import context as otcontext
+from opentelemetry import trace
+from opentelemetry.instrumentation.logging import LoggingInstrumentor
+from settings_library.tracing import TracingSettings
+
+TracingContext: TypeAlias = otcontext.Context | None
+
+
+def _is_tracing() -> bool:
+    return trace.get_current_span().is_recording()
+
+
+def get_context() -> TracingContext:
+    if not _is_tracing():
+        return None
+    return otcontext.get_current()
+
+
+def attach_context(context: TracingContext) -> None:
+    if context is not None:
+        otcontext.attach(context)
+
+
+def detach_context(context: TracingContext) -> None:
+    if context is not None:
+        otcontext.detach(context)
+
+
+def setup_log_tracing(tracing_settings: TracingSettings):
+    _ = tracing_settings
```

Comment:
TIP: here you want to comply with the interface but you actually do not need the input. Do at least something useful with it e.g. 
```python
assert tracing_settings.TRACING_OPENTELEMETRY_COLLECTOR_ENDPOINT # nosec
``

Reply:
I believe this will always evaluate to true. Because the endpoint is a url, so I don't think it can be empty. Then I would rather leave out the input entirely.